ddtrace/contrib/cassandra/session.py

```diff
@@ -166,14 +105,46 @@ def _extract_result_metas(result):
 
     return metas
 
-def _sanitize_query(query):
+def _sanitize_query(span, query):
     """ Sanitize the query to something ready for the agent receiver
     - Cast to unicode
     - truncate if needed
     """
     # TODO (aaditya): fix this hacky type check. we need it to avoid circular imports
-    if type(query).__name__ in ('SimpleStatement', 'PreparedStatement'):
+    t = type(query).__name__
+
+    resource = None
+    if t in ('SimpleStatement', 'PreparedStatement'):
         # reset query if a string is available
-        query = getattr(query, "query_string", query)
+        resource = getattr(query, "query_string", query)
+    elif t == 'BatchStatement':
+        resource = 'BatchStatement'
+        q = "; ".join(q[1] for q in query._statements_and_parameters[:2])
+        span.set_tag("cassandra.query", q)
+        span.set_metric("cassandra.batch_size", len(query._statements_and_parameters))
+    elif t == 'BoundStatement':
+        ps = getattr(query, 'prepared_statement', None)
+        if ps:
+            resource = getattr(ps, 'query_string', None)
+    else:
+        resource = 'unknown-query-type' # FIXME[matt] what else do to here?
```

Comment:
In this way aren't we setting as a resource ``unknown-query-type `` for all queries that aren't in the list above? I mean we're failing tests because of that. Is it the expected behavior?

Failure example:
```
======================================================================
FAIL: tests.contrib.cassandra.test.TestCassPatchAll.test_get_traced_cassandra
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/dd-trace-py/.tox/py34-cassandra/lib/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/ubuntu/dd-trace-py/tests/contrib/cassandra/test.py", line 78, in test_get_traced_cassandra
    eq_(query.resource, self.TEST_QUERY)
nose.proxy.AssertionError: 'unknown-query-type' != 'SELECT * from test.person'
-------------------- >> begin captured logging << --------------------
cassandra.policies: INFO: Using datacenter 'datacenter1' for DCAwareRoundRobinPolicy (via host '127.0.0.1'); if incorrect, please specify a local_dc to the constructor, or limit contact points to local cluster nodes
ddtrace.encoding: DEBUG: using Msgpack encoder
ddtrace.encoding: DEBUG: using Msgpack encoder
ddtrace.encoding: DEBUG: using JSON encoder; application performance may be degraded
ddtrace.encoding: DEBUG: using Msgpack encoder
cassandra.policies: INFO: Using datacenter 'datacenter1' for DCAwareRoundRobinPolicy (via host '127.0.0.1'); if incorrect, please specify a local_dc to the constructor, or limit contact points to local cluster nodes
--------------------- >> end captured logging << ---------------------
```